src/Bicep.Core/PrettyPrintV2/SyntaxLayouts.cs

```diff
@@ -33,7 +33,11 @@ public partial class SyntaxLayouts
                 syntax.Children,
                 syntax.CloseBracket,
                 separator: LineOrCommaSpace,
-                padding: LineOrEmpty);
+                padding: LineOrEmpty,
+                forceBreak:
+                    // If the array contains a newline before the first property, always break the the object.
```

Comment:
nit: "the the"

Reply:
Ah good catch. That's an error in the original comment I copied. Fixed it.